org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/managers/InvisibleProjectBuildSupportTest.java

```diff
@@ -429,4 +437,30 @@ public void testImportReferencedLibrariesConfiguration() throws Exception {
 			assertEquals("Configuration with sources", libraries, preferences.getReferencedLibraries());
 		}
 	}
+
+	@Test
+	public void testUpdateReferencedLibraries() throws Exception {
+		IProject project = copyAndImportFolder("singlefile/simple", "src/App.java");
+		NavigateToDefinitionHandler handler = new NavigateToDefinitionHandler(preferenceManager);
+		String uri = ClassFileUtil.getURI(project, "App");
+		TextDocumentIdentifier identifier = new TextDocumentIdentifier(uri);
+		List<? extends Location> definitions = handler.definition(new TextDocumentPositionParams(identifier, new Position(0, 13)), monitor);
+		
```

Comment:
In the commit viewer, I noticed this line has an extraneous whitespace. I would probably fix this just before merging.